### PR TITLE
Generate url for analysis and allow users to copy/share

### LINKED ISSUE #434: Add link URL for analysis
When a user runs an analysis and clicks "run analysis", we should update the URL to save that information

And when coming from this link, autofill the analysis module.

### PR DESCRIPTION
**Pull request by tdlinh2712**
This PR closes #434 
Deployed at : http://prism-434.surge.sh/

To test this PR:
1. REACT_APP_COUNTRY=cambodia yarn start (or pick any other country) or go to http://prism-434.surge.sh/
2. Run analysis: 10-day rainfall + IDPoor, any threshold/date
3. Click 'run analysis', wait for the result
4. Click on "Share analysis", the URL will be copied to clipboard.
5. Open a new tab and paste the URL to the new tab. You can see that the analysis table is prefilled with the same settings. Click "Run analysis" and wait for the result. The result in the new tab should be the same as the old tab,

### CODE REVIEW

**Comment by tdlinh2712** [comment]
@ericboucher please check my questions in the PR description. thank you

**Comment by tdlinh2712** [comment]
- [ ] Refactor `components/Analyser/index.tsx`
- [ ] Add button for copy search to clipboard
- [ ] Add url to mapview functionality

File: src/components/MapView/Analyser/index.tsx [hunk 1]
```diff
@@ -294,6 +359,18 @@ function Analyser({ extent, classes }: AnalyserProps) {
     }
   };
 
+  const shareAnalysis = () => {
+    copyTextToClipboard(window.location.href).then(() => {
+      // displaySuccessMessage('Copied to clipboard!');
```
Comment (ericboucher) [inline src/components/MapView/Analyser/index.tsx:364]: 👣

**Review by ericboucher — APPROVED** [review]

**Comment by ericboucher** [comment]
Looks like the thresholds get inverted when parsing the url

**Comment by ericboucher** [comment]
@wadhwamatic could you give this a quick review? Thanks!

**Comment by wadhwamatic** [comment]
This is great, thanks! I added a missing translation for the 'share analysis' button in Mozambique. 

I think it would be worthwhile to add a message when a user first lands on an analysis page to let them know some actions is needed. For example: "Analysis configuration has been loaded, click 'Run analysis' to execute it". Let me know if you agree and if you want to add that now, or in a separate PR

**Review by wadhwamatic — APPROVED** [review]

**Pull request merged**